Artifacts/windows-vscode/Artifactfile.json

```diff
@@ -8,9 +8,9 @@
         "VSCode",
         "Visual Studio"
     ],
-    "iconUri": "https://github.com/Azure/azure-devtestlab/raw/master/Artifacts/windows-vscode/vscode%20icon.png",
+    "iconUri": "https://github.com/Azure/azure-devtestlab/raw/master/Artifacts/windows-vscode/vscode.png",
     "targetOsType": "Windows",
     "runCommand": {
-        "commandToExecute": "powershell.exe -ExecutionPolicy bypass -File installVSCode.ps1"
+        "commandToExecute": "powershell.exe -ExecutionPolicy bypass \"& ./install-vscode.ps1\""
```

Comment:
Out of curiosity, why use "& ./install-vscode.ps1" instead of "-File .\install-vscode.ps1" ?

Reply:
To bubble up the errors correctly. `-File` doesn't allow exit codes to fail the call.